src/onegov/search/integration.py

```diff
@@ -394,8 +398,24 @@ def reindex_model(model):
 
         self.es_indexer.bulk_process()
 
+    def psql_perform_reindex(self, request):
+        """ Re-indexes all `searchable' models in postgresql db ensuring
+        each table will be indexed only once.
+
+        """
+        done = []
+
+        for model in searchable_sqlalchemy_models(Base):
+            if model.__tablename__ not in done:
+                model.reindex(request, model)
+                done.append(model.__tablename__)
+
+    def psql_search(self, query=''):
+        from onegov.org.models import SearchPostgres
+        return SearchPostgres(self.request(), query).generic_search()
```

Comment:
Please get rid of this dependency on `onegov.org` from `onegov.search`.

`onegov.search` is a more generic module that should work with or without `onegov.org`. If you want this method to be accessible through `psql_search` then you should either move `SearchPostgres` to `onegov.search` or extract the functionality you want into `onegov.search` and use it in both places.

`SearchPostgres` does not appear to contain any dependency on `onegov.org` itself, so there's no good reason to put it there anyways, besides the original `Search` pagination also being there.

Reply:
removed